README.md

```diff
@@ -72,89 +74,97 @@ module "vpc" {
+| --------------------- | ---------------------------------------------------------------- | :----: | :-----: | :------: |
+| subnet_name           | The name of the subnet being created                             | string |    -    |    yes   |
+| subnet_ip             | The IP and CIDR range of the subnet being created                | string |    -    |    yes   |
+| subnet_region         | The region where the subnet will be created                      | string |    -    |    yes   |
+| subnet_private_access | Whether this subnet will have private Google access enabled      | string |  false  |    no    |
+| subnet_flow_logs      | Whether the subnet will record and send flow log data to logging | string |  false  |    no    |
+| description           | The description of the subnet being created                      | string |   null  |    no    |
 
 ### Route Inputs
-The routes list contains maps, where each object represents a route. For the next_hop_* inputs, only one is possible to be used in each route. Having two next_hop_* inputs will produce an error. Each map has the following inputs (please see examples folder for additional references):
-
-| Name | Description | Type | Default | Required |
-|------|-------------|:----:|:-----:|:-----:|
-| name | The name of the route being created  | string | - | no |
-| description | The description of the route being created | string | - | no |
-| tags | The network tags assigned to this route. This is a list in string format. Eg. "tag-01,tag-02"| string | - | yes |
-| destination_range | The destination range of outgoing packets that this route applies to. Only IPv4 is supported | string | - | yes
-| next_hop_internet | Whether the next hop to this route will the default internet gateway. Use "true" to enable this as next hop | string | - | yes |
-| next_hop_ip | Network IP address of an instance that should handle matching packets | string | - | yes |
-| next_hop_instance |  URL or name of an instance that should handle matching packets. If just name is specified "next_hop_instance_zone" is required | string | - | yes |
-| next_hop_instance_zone |  The zone of the instance specified in next_hop_instance. Only required if next_hop_instance is specified as a name | string | - | no |
-| next_hop_vpn_tunnel | URL to a VpnTunnel that should handle matching packets | string | - | yes |
-| priority | The priority of this route. Priority is used to break ties in cases where there is more than one matching route of equal prefix length. In the case of two routes with equal prefix length, the one with the lowest-numbered priority value wins | string | 1000 | yes |
+
+The routes list contains maps, where each object represents a route. For the next_hop** inputs, only one is possible to be used in each route. Having two next_hop** inputs will produce an error. Each map has the following inputs (please see examples folder for additional references):
```

Comment:
It looks like you might have some sort of markdown autoformatter? Could you restrict the change to adding the description field?

Reply:
ahh yeah, thats my IDE, it auto-formats, I'll turn that off and re-commit :)